app/models/post.rb

```diff
@@ -2,6 +2,9 @@ class Post < ApplicationRecord
   belongs_to :author, class_name: 'User'
   has_many :likes
   has_many :comments
+  validates :title, presence: true, length: { maximum: 250 }
```

Comment:
- [OPTIONAL] Great job so far 👏🏻  what if we add a validation mesage in case the maximum title length is greating than 250 👍🏻 
- eg: validates :title, presence: true,
                   length: { maximum: 250, too_long: 'Name must not exceed 250 characters' }